lib/Element.js

```diff
@@ -233,6 +237,14 @@ Element.prototype.getElementById = function (id) {
   }
 }
 
+Element.prototype.getBoundingClientRect = function () {
+  if (!this.component) {
+    return undefined
+  }
+
+  return this.component.getBoundingClientRect()
+}
```

Comment:
Thanks for your reply. "The bounding rect is something computed after the browser has rendered the element" <- Yes, that is correct. This is working for me (i.e. it returns the real DOM element's ClientRect) just as soon as React has mounted the element to the DOM. In Element's constructor, I added a ref property, which gets added to the React element as its ref attribute... after mounting, React passes the DOM element to the callback function which sets it as a property on the Element instance. Anyway to unit test, I think I'll need to fake the DOM... Any suggestions?